.github/workflows/prerelease_linux_on_demand.yml

```diff
@@ -6,13 +6,17 @@ on:
       branch:
         description: 'Branch to build from'
         required: true
+      fake_tag:
+        description: 'Version to be given to the packages'
+        default: '0.0.0'
       dest_prefix:
         description: 'Repo prefix'
         required: true
 
 env:
   GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  TAG: "0.0.0"
+  TAG: ${{ inputs.fake_tag }}
+  FAKE_TAG:  ${{ inputs.fake_tag }}
```

Comment:
Why having 2, it seems like TAG is not used anymore, is it?

Reply:
it is used by some internals of goreleaser 😕

Reply:
Oks!